cheroot/makefile.py

```diff
@@ -1,21 +1,12 @@
 """Socket file object."""
 
-from __future__ import absolute_import, division, print_function
-__metaclass__ = type
-
 import socket
 
-try:
-    # prefer slower Python-based io module
-    import _pyio as io
-except ImportError:
-    # Python 2.6
-    import io
-
-import six
+# prefer slower Python-based io module
```

Comment:
@jaraco I'm not sure if this is really necessary in 2021. WDYT?

Reply:
Agreed it's probably not necessary, but I intentionally left it out of scope for this change, as I don't know what the implications might be.

Reply:
Fair enough. That's what I thought.